client/src/tests/procedures_tests.rs

```diff
@@ -3,24 +3,25 @@
 
 #![allow(non_snake_case)]
 
-use riker::actors::ActorSystem;
+// use actix::Actor;
+// use riker::actors::ActorSystem;
```

Comment:
Cleanup the comments from the old imports.  No need to keep the riker stuff since its gone.